docs/reference/node-chart.md

```diff
@@ -56,36 +56,37 @@ graph LR
   Device -.-> Skin[[Skin](skin.md)]
   Device -.-> SolidDevice(Solid Device)
 
-  Group[[Group](group.md)] --> Transform[[Transform](transform.md)]
+  Group[[Group](group.md)] --> Pose[[Pose](pose.md)]
   Group --> Billboard[[Billboard](billboard.md)]
-    Transform --> Solid[[Solid](solid.md)]
-      Solid --> SolidDevice
-        SolidDevice --> Accelerometer[[Accelerometer](accelerometer.md)]
-        SolidDevice --> Altimeter[[Altimeter](altimeter.md)]
-        SolidDevice --> Camera[[Camera](camera.md)]
-        SolidDevice --> Compass[[Compass](compass.md)]
-        SolidDevice --> Connector[[Connector](connector.md)]
-        SolidDevice --> Display[[Display](display.md)]
-        SolidDevice --> DistanceSensor[[DistanceSensor](distancesensor.md)]
-        SolidDevice --> Emitter[[Emitter](emitter.md)]
-        SolidDevice --> GPS[[GPS](gps.md)]
-        SolidDevice --> Gyro[[Gyro](gyro.md)]
-        SolidDevice --> InertialUnit[[InertialUnit](inertialunit.md)]
-        SolidDevice --> LED[[LED](led.md)]
-        SolidDevice --> Lidar[[Lidar](lidar.md)]
-        SolidDevice --> LightSensor[[LightSensor](lightsensor.md)]
-        SolidDevice --> Pen[[Pen](pen.md)]
-        SolidDevice --> Propeller[[Propeller](propeller.md)]
-        SolidDevice --> Radar[[Radar](radar.md)]
-        SolidDevice --> RangeFinder[[RangeFinder](rangefinder.md)]
-        SolidDevice --> Receiver[[Receiver](receiver.md)]
-        SolidDevice --> Speaker[[Speaker](speaker.md)]
-        SolidDevice --> TouchSensor[[TouchSensor](touchsensor.md)]
-      Solid --> Track[[Track](track.md)]
-      Solid --> Charger[[Charger](charger.md)]
-      Solid --> Robot[[Robot](robot.md)]
-    Transform --> Fluid[[Fluid](fluid.md)]
-    Transform --> TrackWheel[[TrackWheel](trackwheel.md)]
+  Group --> TrackWheel[[TrackWheel](trackwheel.md)]
```

Comment:
In `TrackWheel.hpp` TrackWheel is derived from Pose

Reply:
Yes in the code of Webots (I guess @ad-daniel implemented like this for convenience), but not for the users because it doesn't have a `translation` field and its `rotation` field is hidden. So, it only inherits the `children` field from `Group`.